op-batcher/metrics/metrics.go

```diff
@@ -243,6 +258,24 @@ func (m *Metrics) RecordChannelClosed(id derive.ChannelID, numPendingBlocks int,
 	m.channelClosedReason.Set(float64(ClosedReasonToNum(reason)))
 }
 
+func (m *Metrics) RecordL2BlockInPendingQueue(block *types.Block) {
+	var size uint64
+	for _, tx := range block.Transactions() {
+		size += tx.Size()
+	}
+	m.pendingBlocksBytesCount.Add(float64(size))
+	m.pendingBlocksBytesCurrent.Add(float64(size))
+}
+
+func (m *Metrics) RecordL2BlockInChannel(block *types.Block) {
+	var size uint64
+	for _, tx := range block.Transactions() {
+		size += tx.Size()
+	}
+	m.pendingBlocksBytesCurrent.Add(-1 * float64(size))
+	// Refer to RecordL2BlocksAdded to see the current + count of bytes added to a channel
+}
```

Comment:
Why did you decide to sum up transaction sizes, instead of the whole block (using rlp-encoded size)? Isn't this slightly less also? The metric would be more comparable to the other `input_bytes` and `input_bytes_total` metrics if you use full rlp block encoding size.

Reply:
FYI `input_bytes` is computed from the actual size of the batches (not the blocks)